utils/grafana.js

```diff
@@ -4,11 +4,8 @@ import { COUNT } from '@/config/types';
 export function computeDashboardUrl(embedUrl, clusterId, params) {
   const url = parseUrl(embedUrl);
   const clusterPrefix = clusterId === 'local' ? '' : `/k8s/clusters/${ clusterId }`;
-  const prefix = `${ clusterPrefix }/api/v1/namespaces/cattle-monitoring-system/services/http:rancher-monitoring-grafana:80/proxy/`;

```

Comment:
I haven't tested this, but could you confirm that this doesn't break rancher metrics?

Reply:
Sure, I have confirmed it. Rancher metrics are working expectly.

![image](https://user-images.githubusercontent.com/18737885/133034404-b25f9960-fe3d-48f3-b7d6-a6c62c0c9a8f.png)